cloudformation/mongo-opsmanager-backup.template

```diff
@@ -16,6 +16,25 @@
         "PROD"
       ]
     },
+    "Stack": {
+      "Description": "Stack name",
+      "Type": "String"
+    },
+    "SshKeyFileS3Url": {
+      "Description": "S3 URL for a list of public SSH keys to use for the default ubuntu user",
+      "Type": "String",
+      "Default": ""
+    },
+    "EBSOptions": {
+      "Description": "Extra parameters to add-encrypted script",
+      "Type": "String",
+      "Default": "-t io1 -i 1000"
```

Comment:
Sounds fair. I'd move to the general purpose volumes (`gp2`). That's what we switched to in CODE and they seem pretty fast and I'd say are a good compromise.